### DISCUSSION TITLE
Allow DevQuest to create DevQuest events

### DISCUSSION THREAD

**Discussion by redwizard42** [RetroAchievements/RAWeb#2910] [Ideas]
https://github.com/RetroAchievements/RAWeb/pull/2864

It would be nice if DevQuest team were able to create a DevQuest event with N achievements authored by DevQuest similar to admins or to have a flag on the even creation so that an admin could direct the event-creation to be authored by DevQuest.

**Comment by Jamiras** [comment]
#2912
